decisions/community-forum-platform.md

```diff
@@ -0,0 +1,349 @@
+
+# Factors to Consider
+
+## Factor: Distinction between Discussion, Q&A and Proposals Toward Action/Creation/Improvement
+
+We need forums for two very different types of communication -- A) Proposals Toward Action/Creation/Improvement and B) Open Discussion and Q&A
+
+Github issues and pull requests have worked brilliantly as a forum for Proposals Toward Action/Creation/Improvement, where someone proposes a feature, product, bug fix, documentation change, etc and people discuss the proposal to decide what action should be taken. If the action is completed, the issue gets closed. If the action isn't completed, the issue remains open.
+
+By contrast, we still haven't found the right combination of tools & habits to support Open Discussion and Q&A, where threads of discussion can stay open permanently, with their content evolving over months without getting lost or going stale.
+
+## Factor: Scaling to a Worldwide Community with Thousands of Users
+
+There are a number of factors that directly impact our ability to scale to a worldwide community with thousands of users. Some of those factors are:
+
+- [Relying heavily on the contributors creates a bottleneck](#scaling-factor-relying-heavily-on-the-core-contributors-creates-a-bottleneck)
+- [Proliferation of Comms Channels and Resulting Notification Overload](#scaling-factor-proliferation-of-comms-channels-and-resulting-notification-overload)
+- [Multilingual Support](#scaling-factor-multilingual-support)
+- [Time Zones and Asynchronous Communication](#scaling-factor-time-zones-and-asynchronous-communication)
+
+### Scaling Factor: Relying heavily on the core contributors creates a bottleneck
+
+The core contributors are already maxed out with communications that are directly related to ongoing work. While we can improve their workflow and workloads to reduce this burden, we will never reach a truly global scale if we continue to route the majority of community discussions through the core contributors.
+
+Worse, if we don't fix this bottleneck it will actually limit the number of core contributors because potential contributors will be driven away, one by one, when they give up due to frustration, lack of information, or lack of encouragement.
+
+In order to bypass this bottleneck, we need to provide infrastructure and documentation that will allow community members to directly support and inform each other while maintaining a high signal-to-noise ratio in whichever communication forums they use. The core contributors will always have a huge role to play in this, but they can't and shouldn't be the only conduit of high-quality information.
+
+### Scaling Factor: Proliferation of Comms Channels and Resulting Notification Overload
+
+Spreading communication across many  channels confuses the heck out of everyone, especially new arrivals. It also makes it very hard for core contributors to participate in open community discussions on top of the existing volume of implementation-focused discussions.
```

Comment:
I feel a tool like Discourse could decrease cognitive load on core devs. 

At Github,  they can watch `ipfs/faq` or `ipfs/community` but this means they will receive a notification for EVERY message. If they are not interested in issue, they need to explicitly **opt-out**. 

In Discourse there is a setting that sends notification for the first post in thread, and it is up to dev to decide if he or she wants to follow it further (**opt-in**):

> ![2017-03-14 08_48_24-latest development topics - mozilla discourse](https://cloud.githubusercontent.com/assets/157609/23890672/10afbe02-0893-11e7-9afe-88fc4209c456.png)